assets/styles/style.css

```diff
@@ -85,279 +108,388 @@ IMAGE GALLERY STYLES
-			.respond-form form input[type=email]:focus,
-			.respond-form form input[type=url]:focus,
-			.respond-form form textarea:focus {
-				background: #fff; }
-			.respond-form form input[type=text]:invalid,
-			.respond-form form input[type=email]:invalid,
-			.respond-form form input[type=url]:invalid,
-			.respond-form form textarea:invalid {
-				outline: none;
-				border-color: #fbc2c4;
-				background-color: #f6e7eb;
-				-ms-box-shadow: none;
-				box-shadow: none; }
-		.respond-form form input[type=text],
-		.respond-form form input[type=email],
-		.respond-form form input[type=url] {
-			max-width: 400px;
-			min-width: 250px; }
-		.respond-form form textarea {
-			resize: none;
-			width: 97.3%;
-			height: 150px; }
+	padding-bottom: 2.2em;
+}
+
+.respond-form form {
+	margin: 0.75em 0;
+}
+
+.respond-form form li {
+	list-style-type: none;
+	clear: both;
+	margin-bottom: 0.7335em;
+}
+
+.respond-form form li label,
+.respond-form form li small {
+	display: none;
+}
+
+.respond-form form input[type=text],
+.respond-form form input[type=email],
+.respond-form form input[type=url],
+.respond-form form textarea {
+	padding: 3px 6px;
+	background: #efefef;
+	border: 2px solid #cecece;
+	line-height: 1.5em;
+}
+
+.respond-form form input[type=text]:focus,
+.respond-form form input[type=email]:focus,
+.respond-form form input[type=url]:focus,
+.respond-form form textarea:focus {
+	background: #fff;
+}
+
+.respond-form form input[type=text]:invalid,
+.respond-form form input[type=email]:invalid,
+.respond-form form input[type=url]:invalid,
+.respond-form form textarea:invalid {
+	outline: none;
+	border-color: #fbc2c4;
+	background-color: #f6e7eb;
+	-ms-box-shadow: none;
+	box-shadow: none;
+}
+
+.respond-form form input[type=text],
+.respond-form form input[type=email],
+.respond-form form input[type=url] {
+	max-width: 400px;
+	min-width: 250px;
+}
+
+.respond-form form textarea {
+	resize: none;
+	width: 97.3%;
+	height: 150px;
+}
 
 #comment-form-title {
-	margin: 0 0 1.1em; }
+	margin: 0 0 1.1em;
+}
 
 #allowed_tags {
-	margin: 1.5em 10px 0.7335em 0; }
+	margin: 1.5em 10px 0.7335em 0;
+}
 
 .nocomments {
-	margin: 0 20px 1.1em; }
+	margin: 0 20px 1.1em;
+}
 
 /*********************
 SIDEBARS & ASIDES
 *********************/
 .sidebar li {
-	list-style: none; }
+	list-style: none;
+}
 
 .widget ul {
-	margin: 0; }
+	margin: 0;
+}
 
 /*********************
 FOOTER STYLES
 *********************/
 .footer {
 	clear: both;
-	margin-top: 1em; }
+	margin-top: 1em;
+}
 
 /*********************
 VISUAL EDITOR STYLES
 *********************/
 body#tinymce {
-	margin: 20px; }
+	margin: 20px;
+}
 
 /*********************
 PLUGIN STYLES
 *********************/
 .gform_body ul {
 	list-style: none outside none;
-	margin: 0; }
-
-/*# sourceMappingURL=style.css.map */
+	margin: 0;
+}
 
 /*********************/
 /*    BFCom styles   */
 /*********************/
+/* PAGE AND LAYOUT */
+body {
+	font-family: Pangram,Avenir,Helvetica,sans-serif;
```

Comment:
One challenge with using fonts that the user might not have is that it'll make the layout sizing less predictable. Maybe you and I have Pangram installed, so a word is X pixels wide, whereas someone else only has Helvetica, and the same word is Y pixels wide.

Of course hopefully our designs aren't that sensitive to the width of specific words.

One solution is to use a web font, of course.

Reply:
I'd be happy to go with whatever works best. That font-family was copied from medium.com. I like the look but don't feel wedded to the particular fonts.

Reply:
Cool; well, let's roll with it and see how it goes.